fantasybot/telegram/feedback.py

```
@@ -0,0 +1,46 @@
+def load_all_feedback() -> list:
+    if not os.path.exists(FEEDBACK_PATH):
+        return []
+    results = []
+    with open(FEEDBACK_PATH, "r", encoding="utf-8") as f:
+        for line in f:
+            line = line.strip()
+            if line:
+                try:
+                    results.append(json.loads(line))
+                except Exception:
+                    pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)